README.md

```diff
@@ -173,6 +173,9 @@ XHP supports Hack's 'async' functionality, allowing you to build components that
 efficiently fetch the data that they require:
 
 ```hack
+// @fredemmott I think you changed some of the function names here.
+// Could you please validate that this example is still correct?
+// I think we might want to use `xhp class` here.
```

Comment:
Should have been a GitHub comment instead :)

Reply:
Oof